### PR TITLE
Add windicss example

### PR DESCRIPTION
**Pull request by AviAvinav**
Adding windicss example.

This is my first pull request to next.js, so I am very sorry if I did something wrong.

## Documentation / Examples

- [X] Make sure the linting passes by running `yarn lint`

### CODE REVIEW

File: examples/with-typescript-windicss/.gitignore [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+.env.local
+.env.development.local
+.env.test.local
+.env.production.local
+
+# vercel
+.vercel
+
+# typescript
+*.tsbuildinfo
```
Comment (balazsorban44) [inline examples/with-typescript-windicss/.gitignore:38]:
```suggestion
.env*.local

# vercel
.vercel

# typescript
*.tsbuildinfo
next-env.d.ts
```

File: examples/with-typescript-windicss/package.json [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+  "scripts": {
+    "dev": "next dev",
+    "build": "next build",
+    "start": "next start",
+    "lint": "next lint"
+  },
+  "dependencies": {
+    "next": "12.1.0",
+    "react": "17.0.2",
+    "react-dom": "17.0.2"
```
Comment (balazsorban44) [inline examples/with-typescript-windicss/package.json:12]:
```suggestion
    "next": "latest",
    "react": "18.2.0",
    "react-dom": "18.2.0"
```

File: examples/with-typescript-windicss/package.json [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+  },
+  "devDependencies": {
+    "@types/node": "17.0.21",
+    "@types/react": "17.0.40",
+    "eslint": "8.10.0",
+    "eslint-config-next": "12.1.0",
```
Comment (balazsorban44) [inline examples/with-typescript-windicss/package.json:18]:
```suggestion
```

File: examples/with-typescript-windicss/package.json [hunk 4]
```diff
@@ -0,0 +1,23 @@
+{
+  "private": true,
+  "scripts": {
+    "dev": "next dev",
+    "build": "next build",
+    "start": "next start",
+    "lint": "next lint"
```
Comment (balazsorban44) [inline examples/with-typescript-windicss/package.json:7]:
```suggestion
    "start": "next start"
```

File: examples/with-typescript-windicss/README.md [hunk 5]
```diff
@@ -0,0 +1,15 @@
...
+This example shows how to use [WindiCSS](https://windicss.org/) with Next.js.
+
+## How to use
+
+Execute [`create-next-app`](https://github.com/vercel/next.js/tree/canary/packages/create-next-app) with [npm](https://docs.npmjs.com/cli/init) or [Yarn](https://yarnpkg.com/lang/en/docs/cli/create/) to bootstrap the example:
+
+```bash
+npx create-next-app --example with-typescript-windicss-app
+# or
+yarn create next-app --example with-typescript-windicss-app
```
Comment (balazsorban44) [inline examples/with-typescript-windicss/README.md:12]:
```suggestion
npx create-next-app --example with-windicss
# or
yarn create next-app --example with-windicss
# or
pnpm create next-app --example with-windicss
```

File: examples/with-typescript-windicss/README.md [hunk 6]
```diff
@@ -0,0 +1,17 @@
+# Next.js + WindiCSS Example
+
+This example shows how to use [WindiCSS](https://windicss.org/) with Next.js.
+
```
Comment (balazsorban44) [inline examples/with-typescript-windicss/README.md:4]:
```suggestion

## Deploy your own

Deploy the example using [Vercel](https://vercel.com?utm_source=github&utm_medium=readme&utm_campaign=next-example) or preview live with [StackBlitz](https://stackblitz.com/github/vercel/next.js/tree/canary/examples/hello-world)

[![Deploy with Vercel](https://vercel.com/button)](https://vercel.com/new/git/external?repository-url=https://github.com/vercel/next.js/tree/canary/examples/with-windicss&project-name=with-windicss&repository-name=with-windicss)

```

**Review by balazsorban44 — CHANGES_REQUESTED** [review]
Hi @AviAvinav could we rename this folder to `examples/with-windicss`? After that, we could merge the PR!

**Comment by AviAvinav** [comment]
Yeah, sure

**Comment by AviAvinav** [comment]
@balazsorban44 renamed it, please review

**Review by balazsorban44 — APPROVED** [review]
Thanks!

**Pull request merged**